x-pack/plugins/observability/public/pages/cases/empty_page.tsx

```diff
@@ -0,0 +1,118 @@
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  IconType,
+  EuiCard,
+} from '@elastic/eui';
+import React, { MouseEventHandler, ReactNode, useMemo } from 'react';
+import styled from 'styled-components';
+
+const EmptyPrompt = styled(EuiEmptyPrompt)`
+  align-self: center; /* Corrects horizontal centering in IE11 */
+  max-width: 60em;
+`;
+
+EmptyPrompt.displayName = 'EmptyPrompt';
+
+interface EmptyPageActions {
+  icon?: IconType;
+  label: string;
+  target?: string;
+  url: string;
+  descriptionTitle?: string;
+  description?: string;
+  fill?: boolean;
+  onClick?: MouseEventHandler<HTMLButtonElement | HTMLAnchorElement>;
+}
+
+export type EmptyPageActionsProps = Record<string, EmptyPageActions>;
+
+interface EmptyPageProps {
+  actions: EmptyPageActionsProps;
+  'data-test-subj'?: string;
+  message?: ReactNode;
+  title: string;
+}
+
+const EmptyPageComponent = React.memo<EmptyPageProps>(({ actions, message, title, ...rest }) => {
+  const titles = Object.keys(actions);
+  const maxItemWidth = 283;
+  const renderActions = useMemo(
+    () =>
+      Object.values(actions)
+        .filter((a) => a.label && a.url)
+        .map(
+          (
+            { icon, label, target, url, descriptionTitle, description, onClick, fill = true },
+            idx
+          ) =>
+            descriptionTitle != null || description != null ? (
+              <EuiFlexItem
+                grow={false}
+                style={{ maxWidth: maxItemWidth }}
+                key={`empty-page-${titles[idx]}-action`}
+              >
+                <EuiCard
+                  title={descriptionTitle ?? false}
+                  description={description ?? false}
+                  footer={
+                    /* eslint-disable-next-line @elastic/eui/href-or-on-click */
+                    <EuiButton
+                      href={url}
+                      onClick={onClick}
+                      iconType={icon}
+                      target={target}
+                      fill={fill}
+                      data-test-subj={`empty-page-${titles[idx]}-action`}
+                    >
+                      {label}
+                    </EuiButton>
+                  }
+                />
+              </EuiFlexItem>
+            ) : (
+              <EuiFlexItem
+                grow={false}
+                style={{ maxWidth: maxItemWidth }}
+                key={`empty-page-${titles[idx]}-action`}
+              >
+                {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
+                <EuiButton
+                  href={url}
+                  onClick={onClick}
+                  iconType={icon}
+                  target={target}
+                  data-test-subj={`empty-page-${titles[idx]}-action`}
+                >
+                  {label}
+                </EuiButton>
+              </EuiFlexItem>
+            )
+        ),
+    [actions, titles]
+  );
+
+  return (
+    <EmptyPrompt
+      iconType="logoSecurity"
```

Comment:
think this should be `logoObservability` now?

Reply:
good catch